security/vertx-jwt/src/test/java/io/quarkus/ts/security/vertx/AbstractCommonIT.java

```diff
@@ -80,6 +80,8 @@ public void teardown() {
                 .delete("/replicant/" + replicant.getId())
                 .then()
                 .statusCode(anyOf(is(204), is(404)));
+
+        Vertx.vertx().close();
```

Comment:
Same here and everywhere else, please read comment above.

Reply:
Yeah, I have modified the changes accordingly. Please take time to review it and let me know for further modifications.

Thanks !!!